data/korath/korath.txt

```diff
@@ -345,6 +385,59 @@ fleet "Korath Defense"
 		"'olofez" 4
 		"'olofez (Expeller)" 2
 
+fleet "Korath Humanika Tribute Response"
```

Comment:
This is just a terrible name.
There is already a planet named `Humanika`.
But this fleet has nothing to do with that.

Why has this faction of human pirates(?) named themselves after the Quarg word for humans?

Does this go back to the thing where the Korath use the Quarg word (even though the Kor Efreti, who have had much closer interactions with the Quarg recently, say `Humani Efreti`)?

Reply:
This is intentional, to draw the player's attention to the use of the word by two seemingly unrelated species. "Humanika" means humans in the languages of the Quarg, Efreti, and Exiles. Humani is human (singular) in all three languages. This apparent coincidence will feature prominently in the Korath translator story.

Reply:
In case you didn't notice, the Korath use the word "Humanika" in the first contact, in all routes through the dialog.

Reply:
Also, these "pirates" are definitively not pirates, they are not a faction, as they have no central government, and they have not named themselves "Humanika." The word "Humanika," meaning "humans," is used in the txt for the governments of Korath-associated human groups. The groups do not use it to describe themselves, because they a loose association, not a central government. In game, it shows up as "Independent."

Reply:
**I've added a note about this in "Other minor changes" in the PR description.** I was waiting to see who would be the first to leave a belittling diatribe.